www/js/diary/infinite_scroll_filters.js

```diff
@@ -17,7 +17,7 @@ angular.module('emission.main.diary.infscrollfilters',[
     var unlabeledCheck = function(t) {
        return ConfirmHelper.INPUTS
            .map((inputType, index) => !angular.isDefined(t.userInput[inputType]))
-           .reduce((acc, val) => acc && val, true);
+           .reduce((acc, val) => acc || val, false);
```

Comment:
I assume this is to ensure that users have all labels filled out before we consider the trip as labeled. I think I had it the other way around because I didn't anticipate that users would leave some labels blank. But we did see this case in Ft. Collins, so this is a good fix.

Reply:
Yep!